### DISCUSSION TITLE
Issues with Verifying Academic Affiliation After ID Card Submission

### DISCUSSION THREAD

**Discussion by NishantKB** [community/community#137512] [GitHub Education]
### Hi everyone,

I submitted my ID card to verify my academic affiliation on GitHub, but I received a message saying it couldn’t be verified. I’m now being asked to use my camera to resubmit my documents but in mobile it opens front camera and the quality of my front camera is so low that it cant read the letters on my id card can anyone help me resolve this issue?
![Screenshot 2024-09-02 210019](https://github.com/user-attachments/assets/d711dfe8-1c33-416d-b2c7-6a66a8e833a1)

**Comment by alx-mp4** [comment]
Hi @NishantKB, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!

**Reply by NishantKB** [reply]
![image](https://github.com/user-attachments/assets/35ef43db-375b-41f4-928c-608982724b2b)
